lib/services.template.ejs

```diff
@@ -9,17 +10,18 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' && module.ex
   module.exports = <%-: moduleName | q %>;
 }
```

Comment:
However, the indenting is not so important to fix now (we can do those when we actually modify that line of code.